tests/test_api.rs

```diff
@@ -9,6 +9,7 @@ use std::sync::Mutex;
 
 use rusty_v8 as v8;
 use v8::InIsolate;
+use v8::MapFnTo;
```

Comment:
It's not ideal that the user has to import this "internal" trait. I try to avoid it as much as possible, and I believe I can eventually make it go away.

Reply:
Indeed. Maybe add a TODO comment here reflecting that, so we have a record of the intention.